e2e/rbd_helper.go

```diff
@@ -952,3 +966,22 @@ func waitToRemoveImagesFromTrash(f *framework.Framework, poolName string, t int)
 
 	return err
 }
+
+// getKernelVersionFromContainer gets the kernel version from the specified container.
+func getKernelVersionFromContainer(f *framework.Framework, ns, dsn, cn string) (string, error) {
```

Comment:
it should have been`getKernelVersionFromRbdplugin`.

Reply:
Ah! I should have moved it to utils?

Reply:
or rather into e2e/pod.go?

Reply:
Done now, thanks @Madhu-1